docsite-css/documentation.css

```diff
@@ -65,36 +68,48 @@
 }
 
```

Comment:
If possible remove the empty classes.

Reply:
Didn't get which empty class you're referring to.